build/relayers/relay-on-mainnet.md

```diff
@@ -1,3 +1,329 @@
+
+In addition to relaying packets, this relayer can open paths across chains, thus creating clients, connections and channels.
+
+Additional information on how IBC works can be found  [here](https://ibc.cosmos.network/).
+
+## Hardware Requirements
+
+- **Minimal:**
+	- 4 GB RAM
+	- 50 GB SSD
+	- 2 CPU
+
+- **Recommended:**
+	- 8 GB RAM
+	- 100 GB SSD
+	- 4 CPU
+
+- **Operating System:**
+	- **Recommended:** Linux(x86\_64)
+	- **Others:** Windows/MacOS(x86)
+
+## Prerequisites
+To successfully run a Cosmos Go Relayer, we need to install some prerequirements. Because running the Cosmos Go Relayer software depends on them, prerequirements are also known as dependencies.
+
+We need to install and setup one dependency - **Go**.
+
+### Install Go
+1. Remove any previous installation: 
+	```bash
+	rm -rf /usr/local/go
+	```
+2. Make sure you're installing the latest **Go** version by visiting [this page](https://go.dev/doc/install)
+3. Download the latest version of **Go** (1.19.4 as of time of writing):
+	```bash
+	wget https://go.dev/dl/go1.19.4.linux-amd64.tar.gz
+	```
+4. Extract the contents of the archive into /usr/local: 
+	```bash
+	tar -C /usr/local -xzf go1.19.4.linux-amd64.tar.gz
+	```
+5. Check **Go** is installed correctly *(sample output: `go version go1.19.4 linux/amd64`)*: 
+	```bash
+	go version
+	```
+6. Set $GOPATH:
+
+	1.  Open the `.profile` file, where all your environment variables are stored:
+		```bash
+		nano ~/.profile
+		```
+	2. Scroll down to the end of the file and add the following line before `export $PATH`:
+		```bash
+		export GOPATH=$HOME/go
+		```
+	3. Add the following line to **PATH**  (i.e. `export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin`):
+		```bash
+		$GOPATH/bin
+		```
+	4. Reload the **PATH** environment variable:
+		```bash
+		source ~/.profile
+		```
+	5. Create the directories we set in **PATH**:
+		```bash
+		mkdir -p $GOPATH/bin
+		```
+### Install Git
+1. Install **Git**: 
+	```bash
+	apt install git
+	```
+2. Verify **git** is installed correctly:
+	```bash
+	git --version # Sample Output: git version 2.34.1
+	```
+	
+## Installation Steps
+### 1. Install the Cosmos Relayer Binary
+
+The Cosmos Go Relayer code is available at the [Official GitHub Repository](https://github.com/cosmos/relayer). First, we must 'download' all the code locally by cloning the GitHub repository, choosing the latest version *(as of time of writing: v2.0.0-rc3)* and installing the binary into our GOPATH as explained in the [prerequisites section](#Install-Go) of this guide.
+```bash
+git clone https://github.com/cosmos/relayer.git # Download all the code locally
+cd relayer && git checkout v2.0.0-rc3 # Choose the latest version
```

Comment:
lets stick with latest version, which you have mentioned as `v2.2.0-rc3`

Reply:
I've pushed the change now.